docs/visual-basic/reference/command-line-compiler/langversion.md

```diff
@@ -16,28 +18,31 @@ Causes the compiler to accept only syntax that is included in the specified Visu
 -langversion:version  
 ```  
   
-## Arguments  
- `version`  
- Required. The language version to be used during the compilation. Accepted values are `9`, `10`, `11`, `12`, `14`, `15`, `15.3`, `15.5`, `default` and `latest`.
+## Arguments
+
+ `version`\
+ Required. The language version to be used during the compilation. Accepted values are `9`, `10`, `11`, `12`, `14`, `15`, `15.3`, `15.5`, `16`, `default` and `latest`.
 
  Any of the whole numbers may also be specified using `.0` as the minor version, for example, `11.0`.
```

Comment:
:warning: This currently has a bug with version 16. The compiler seem to accept `16`, but not `16.0`. I opened https://github.com/dotnet/roslyn/pull/46909 to fix that.

Do you want to include a note in this article regarding this?